viscm/gui.py

```diff
@@ -169,10 +190,11 @@ def label(ax, s):
                     verticalalignment="bottom",
                     transform=ax.transAxes)
 
-        Jpapbp = _sRGB1_to_uniform(RGB)
+        Jpapbp = self._sRGB1_to_uniform(RGB)
 
         ax = axes['deltas']
         local_deltas = N * np.sqrt(np.sum((Jpapbp[:-1, :] - Jpapbp[1:, :]) ** 2, axis=-1))
+        print("perceptual delta peak-to-peak: %0.2f" % (np.ptp(local_deltas),))
```

Comment:
Just pushed c65002864b09a2bd4d5f31e3a57cdaf3d6f2c5db which puts a better "flatness" metric directly on the graph